test/updateTree-test.js

```diff
@@ -65,6 +81,14 @@ describe('updateCurrentVelocity', () => {
       .toEqual({a: 0, b: 20.4});
   });
 
+  it('should handle `undefined`', () => {
+    const currentStyle = {a: undefined};
+    const currentVelocity = {a: undefined};
+    const style = {a: undefined};
+    expect(updateCurrentVelocity(FRAME_RATE, currentStyle, currentVelocity, style))
+      .toEqual({a: 0});
```

Comment:
Humm, on second thought this should probably be `undefined`. My fault. Will reconsider this.